README.md

```diff
@@ -19,3 +19,168 @@ files. CMake developers document their functions by immediately proceeding the
 function with a comment block written in reStructuredText (reST). CMakeDoc will
 extract the comment blocks verbatim and create `*.rst` files which can be
 included in a normal Sphinx documentation set-up.
+
+## Installing
+
+#### With PIP / PyPI in Virtual Environment
+Run:
+```console
+foo@bar:~$ mkdir virt-env #Directory for our virtual environment
+foo@bar:~$ python3 -m venv virt-env && cd virt-env #Create our virtual environment and enter the directory
+foo@bar:~/virt-env$ source ./bin/activate #Activate virtual environment
```

Comment:
I'd simplify the first three steps to:

```console
foo@bar:~$ python3 -m venv virt-env #Create a virtual environment
foo@bar:~/virt-env$ source virt-env/bin/activate #Activate virtual environment
```
your instructions actually cause an error with some older versions of Python3 as it'll complain about the directory existing.

Reply:
Fixed in latest commit